static/icons/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Japan COVID-19",
```

Comment:
```suggestion
    "name": "Japan COVID-19 Coronavirus Tracker",
```
Not sure if this is too long?

Reply:
![image](https://user-images.githubusercontent.com/1372136/78526992-b6d13900-7816-11ea-950c-bec27cb004ef.png)

In iOS 13.4, using the full name is too lengthy for icons. I assume it will be the same for Android as well.

Reply:
Understood!